pyschema/core.py

```diff
@@ -410,21 +410,30 @@ def decorator(cls):
     return decorator
 
 
+def identity_value_loader(_, x):
+    return x
+
+
 @no_auto_store()
 class Record(object):
     """Abstract base class for structured logging records"""
     __metaclass__ = PySchema
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, value_loader=identity_value_loader, *args, **kwargs):
```

Comment:
Big no no, value_loader will take up namespace... But we can rename it to __value_loader or something? :)